pkg/smokescreen/smokescreen.go

```diff
@@ -489,10 +490,10 @@ func NormalizeHostPort(hostPort string, forceFQDN bool) (host string, port int,
 // normalized with `normalizeHost` and `normalizePort`.
 //
 // `hostPort` is a bare host or a colon-separated (':') host name and port.
-// If no port is specified, the `scheme`` string is used to find the default
```

Comment:
super minor, but should this and below be a single \` to match the opening \`?

Reply:
I blame VS Code! Cleaned up after it in 2e2e15e and c56faa1.